challenges/objects-arrays.js

```diff
@@ -97,29 +146,43 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+//zooAnimals.forEach( (animal,i) => lowerCase.push(zooAnimals[i].animal_name.toLowerCase()));
+
+const lowerCase = zooAnimals.map( (animal,i) => animal.animal_name.toLowerCase() );
```

Comment:
Looking good! It doesn't look like you're using the second parameter in your callback function, so you could remove it